oddbird/oddsite
# Draft follow-up to smashing conf

## Description

- Blog post about smashing conf!
- video from vueconf
- quotes from Vitaly & Jason & Others

## Steps to test/reproduce

Check out the [netlify branch preview](https://deploy-preview-319--oddsite.netlify.com/), or run it locally, and then…

## Impacted areas of website

- latest post on the "articles" page
- new video on "dynamic css" talk
- new quotes on "dynamic css", "advanced css", and "services/speaking"
